natashaslprice/passion_project.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+
+<head>
+	<link rel = "stylesheet" href = "passion_project.css">
+</head>
+
+<body>
+	<p>I think that the insurance market is a complete scam. People have insurance for pretty much everything, but hardly ever claim because of the time and effort involved. Furthermore, I believe that a lot of people pay for insurance for different things that actually overlap and cover the same items. For example, my scooter insurance covers the clothing I wear on my scooter, which is also covered by my contents insurance. When people travel they may pay for additional travel insurance to cover expensive items, like laptops, which may also be covered in their contents insurance. I would like to model the insurance market and look at whether it is possible to provide people with an all-inclusive insurance product. If I model the market and find out I am correct, then I will be very happy if I can build a website that collects data from people and runs it through a model to provide them with an idea for an insurance product!</p>
```

Comment:
don't let the insurance companies find out about this; they don't want their secret getting out!